packages/thriver-core-styles/lib/styles/browseHappy.import.less

```diff
@@ -0,0 +1,26 @@
+.browsehappy {
+	display:block;
+    position: relative;
+	width:100%;
+    z-index: 27;
+	font-size: 16px;
+	line-height: 16px;
+	color: @gray;
```

Comment:
Please no tab characters (lines 2, 4, 6, 7, and 8)!  You can use a regex search for `\t` and replace all instances with two or four spaces.